fury/actor.py

```diff
@@ -754,6 +754,80 @@ def callback(_caller, _event, calldata=None):
     return actor
 
 
+def dashed_line(p_initial =[0.0, 0.0, 0.0], p_final=[0.0, 0.0, 0.0], 
```

Comment:
Please, do not provide default value for `start_pos` and `end_pos`.

Reply:
Shall I set them as None? or should I leave it completely?